source/lifecycle/mattermost-desktop-releases.md

```diff
@@ -0,0 +1,56 @@
+# Mattermost desktop app releases
+
+```{include} ../_static/badges/allplans-cloud-selfhosted.md
+```
+
+```{Important}
+**Support for Mattermost Server v8.1 Extended Support Release has come to the end of its life cycle on May 15, 2024. Upgrading to Mattermost Server v9.5 Extended Support Release or later is required.**
+```
+
+## Frequency
+
+Mattermost releases a new desktop app version every 4 months, in February, May, August, and November in [binary form](https://docs.mattermost.com/collaborate/install-desktop-app.html#install-and-update-the-mattermost-desktop-app). See the [Desktop app changelog](/lifecycle/desktop-app-changelog) for release details.
+
+
+```{Important}
+- From Mattermost v10, Mattermost server extended releases are now paired with Mattermost desktop app extended releases. See the [Mattermost extended support releases](#extended-support-releases) documentation to learn more about extended releases.
+- Automatic desktop app updates are supported and enabled on Windows, MacOS (via the Apple App Store), Ubuntu, Debian, and generic Linux platforms. See the [install Mattermost desktop app](/collaborate/install-desktop-app.html) documentation for platform-specific details on automatic app updates.
+```
+
+## Latest releases
+
+| **Release** | **Life cycle start** | **Life cycle end** |
```

Comment:
@amyblais - Can I get your help to populate the correct life cycle start and end dates for these desktop releases (based on our desktop changelog content) please?

Reply:
Done for v5.x versions. We can probably remove older versions.

Reply:
Thank you, @amyblais! Open to pulling the older versions out?